src/main/java/reactor/netty/resources/ConnectionProvider.java

```diff
@@ -140,7 +144,7 @@ static ConnectionProvider fixed(String name) {
 	 * number of {@link Connection}
 	 */
 	static ConnectionProvider fixed(String name, int maxConnections) {
-		return fixed(name, maxConnections, DEFAULT_POOL_ACQUIRE_TIMEOUT);
```

Comment:
Keep this and invoke the builder creation just once in `fixed(String name, int maxConnections, long acquireTimeout, @Nullable Duration maxIdleTime, @Nullable Duration maxLifeTime)`